src/posts/features-and-projects.md

```diff
@@ -0,0 +1,146 @@
+
+Next, we export them as SVGs into a font editing software. Here, we adjust the size and spacing of the glyphs, as well as fix font formatting errors (e.g. path self-intersections, missing extremas, path direction issues and non-integral point values). Finally, we export them as .ttf files for use on the sim aircraft’s displays.
+
+The following is a summary of the progress we have made:
+
+- The EIS font has been merged into the aircraft
+- The ISIS font is in the works
+- The DCDU fonts are being reworked
+- The A380 RMP fonts are in advanced progress 
+ 
+Here are some previews of the ISIS and RMP fonts:
+
+![typeface 1](/img/notam-images/march2022/typeface-1.png)
+![typeface 2](/img/notam-images/march2022/typeface-2.png)
+
+One last thing: we rework our fonts as new information and references come, some of which have fundamentally changed our understanding of the displays’ workings (e.g. subpixel array design). As an example, here are photos of our DCDU font before and after reworking:
+
+![typeface 3](/img/notam-images/march2022/typeface-3.png)
+![typeface 4](/img/notam-images/march2022/typeface-4.png)
+
+In this case, it has allowed us to go one step further and reach the final frontier in display typeface replication - previously uncharted territory in the flight simulation industry - sub-pixel accuracy.
+
+### CPDLC with Hoppie
+
+We implemented CPDLC (Controller-Pilot-Datalink-Communication) in the A32NX based on Hoppie's ACARS system. Thereby it is possible to request route clearances on ground (DEPART REQ) and e.g. over the Atlantic (OCEANIC REQ). The whole implementation is based on a realistic realization of the ATS 623 standard. Furthermore, instructions from the air traffic controllers can be received and answered via the DCDU. We took care to display the data in the DCDU as realistically as possible.
+
+![mcdu-atc-comm2](/img/notam-images/march2022/mcdu-atc-comm2.png)
+
+![dcdu-dep-req](/img/notam-images/march2022/dcdu-dep-req.png)
+
+In the next steps, FANS-A and FANS-B systems are implemented for a realistic simulation. Depending on the responsible sector, various communication functions will be activated. The first step is to implement requests that can be sent from the pilot to the air traffic controller. Our goal is a realistic ATSU implementation that also allows automatic responses to requests to be prepared and displayed to the pilot for review and adjustment.
+
+We are also working on a realistic simulation of data link communications based on the VHF data link specification as implemented in real communications.
+This will result in different communication times depending on the other aircraft, available ground stations and distances to the stations. Depending on the situation, this can last from a few seconds to several minutes and is always simulated accordingly.
```

Comment:
```suggestion
This will result in different communication times depending on the other aircraft, available ground stations, and distances to the stations. Depending on the situation, this can last from a few seconds to several minutes and is always simulated accordingly.
```